tests/syndication_tests/feeds.py

```diff
@@ -53,6 +53,22 @@ class TestRss091Feed(TestRss2Feed):
     feed_type = feedgenerator.RssUserland091Feed
 
 
+class TestCategoryObjectRss091Feed(TestRss091Feed):
+    categories = (
+        'python', 78,
+        feedgenerator.Category(
+            'some', label='Something', domain='http://example.com/Some'
+        ),
+        feedgenerator.Category('django', scheme='framework:Django'),
+        feedgenerator.Category(123, domain=b'some-uri', label=3),
+    )
+    item_categories = (
+        'python', 'testing', 11, 12,
+        feedgenerator.Category('simple', domain='http://example.org/Simple'),
+        feedgenerator.Category(b'creativity in tests', scheme=b'bytes here'),
```

Comment:
I'm doubtful that there's a need for bytestring handling considering the sentiment in https://code.djangoproject.com/ticket/27795. Did you have a use case in mind? Could some `force_text()` usage be removed instead?